Automattic/wp-calypso
# Fixes the Blaze credits for atomic sites using the Blaze Dashboard app

Related to SELFSERVE-647

The credits aren't being fetched correctly for atomic sites using the Blaze Dashboard app and appear hidden to the user.

## Proposed Changes

* Fixes the condition in the `useCreditBalanceQuery` to check for a defined value instead of type-checking the site id to number. 

As an additional change, I made minor adjustments to the Blaze Dashboard App documentation and package.json scripts. Now it's easier to test this in sync with the sandbox.

## Testing Instructions

* Open the live preview and go to Tools->Advertising
* Verify that you are able to see the balance for your account

Now, if you like to test the Blaze app, follow the test instructions of [this PR](https://github.com/Automattic/wp-calypso/pull/78703) to set up your local environment.
After that, navigate to Tools->Advertising in your Jurassic tube site and verify that the credits are displayed correctly.

## Pre-merge Checklist
